lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
MSVC expects bytes ([reference](https://msdn.microsoft.com/en-us/library/83ythb65.aspx)).

I thought perhaps there was some other reason for the 64, maybe some magical cache line alignment optimisation? Happy to change both to use 8 byte alignment if you are.